docs/audit-log-filter-variables.md

```diff
@@ -0,0 +1,745 @@
+| Dynamic | Yes  |
+| Scope | Global  |
+| Data type | Integer |
+| Default | 1GB  |
+
+Performs an automatic log file rotation based on the size. The default value is 0 (zero).
+
+If the value is 1GB, the plugin does not automatically rotate the log file. If you set the value to less than 4096 resets the value to 0, and does not automatically rotate the log files. You can rotate the log files manually. The recommended way is to use [`audit_log_rotate()`](#audit_log_rotate).
+
+If the value is greater than 0, when the log file size exceeds the value, the plugin renames the current file and opens a new log file using the original name.
+
+If the value is not a multiple of 4096, the plugin truncates the value to the nearest multiple.
+
+
+### `audit_log_filter_strategy`
+
+| Option name | Description |
+| --- | --- |
+| Command-line | --audit-log-filter-strategy |
+| Dynamic | No  |
+| Scope | Global  |
+| Data type | Enumeration  |
+| Default | ASYNCHRONOUS  |
+
+Defines the Audit Log filter plugin's logging method. The valid values are the following:
+
+| Values | Description |
+| --- | --- |
+| ASYNCHRONOUS | Waits until there is outer buffer space |
+| PERFORMANCE | If the outer buffer does not have enough space, drops requests |
+| SEMISYNCHRONOUS | Operating system permits caching |
+| SYNCHRONOUS | Each request calls `sync()` |
+
+### `audit_log_filter_syslog_tag`
+
+| Option       | Description                     |
+|--------------|---------------------------------|
+| Command-line | --audit-log-filter-syslog-tag= |
+| Dynamic      | No                              |
+| Scope        | Global                          |
+| Data type    | String                         |
+| Default      | audit-filter                         |
+
+### `audit_log_filter_syslog_facility`
+
+| Option name | Description |
+| --- | --- |
+| Command-line | --audit-log-filter-syslog-facility |
+| Dynamic | No  |
+| Scope | Global  |
+| Data type | String  |
+| Default | LOG_USER |
+
+Specifies the syslog `facility` value. The option has the same meaning as the appropriate parameter described in the [syslog(3) manual](https://linux.die.net/man/3/syslog).
+
+
+### `audit_log_filter_syslog_priority`
+
+| Option name | Description |
+| --- | --- |
+| Command-line | --audit-log-filter-syslog-priority |
+| Dynamic | No  |
+| Scope | Global  |
+| Data type | String  |
+| Default | LOG_INFO |
+
+Defines the `priority` value for the syslog. The option has the same meaning as the appropriate parameter described in the [syslog(3) manual](https://linux.die.net/man/3/syslog).
+
+
+
+## Audit log filter status variables
+
+The audit log filter plugin exposes status variables. These variables provide information on the operations.
+
+| Name | Description |
+| -- | --- |
+| `audit_log_filter_buffer_bypassing_writes` | Identifies when the `log_strategy_type` = ASYNCHRONOUS and messages bypass the write buffer and are written directly to the log file  |
```

Comment:
Please rename this one to 'Audit_log_filter_direct_writes'. We renamed it due to recent upstream changes. Logic stays without changes.